lgn-provers/src/lib.rs

```diff
@@ -1,3 +1,16 @@
 #![feature(generic_const_exprs)]
 pub mod params;
 pub mod provers;
+
+#[cfg(feature = "dummy-prover")]
+mod dummy_utils {
+    use std::thread::sleep;
+    use std::time::Duration;
+
+    /// Generates random data to be used as a dummy proof.
+    pub fn dummy_proof(proof_size: usize, sleep_duration: Duration) -> Vec<u8> {
+        sleep(sleep_duration);
```

Comment:
Can we just remove the `sleep`?

Reply:
I didn't know why it was there, so I tried not to change the behaviour. Removing it.

Reply:
IMHO it makes sense to remove it, so that we can use the dummy prover to stress test the GW.